mne/channels/_dig_montage_utils.py

```diff
@@ -187,30 +187,36 @@ def _foo_get_data_from_dig(dig):
     )
 
 
-def _get_fid_coords(dig):
-    fid_coords = dict()
-    fid_coord_frames = []
+def _get_fid_coords(dig, raise_error=True):
```

Comment:
https://github.com/mne-tools/mne-python/pull/6693/files#diff-ca851c003c3eeeaee912998ef5e3e0faR1260 This might also show that maybe we could make use of a public function returning fiducials from Digitizations

Reply:
somehow we have all this `_get_fid_coord(dig)`, `_get_dig_eeg(dig)`, `_count_poinits_by_type(dig)` functions.